btk/catalog.py

```diff
@@ -70,33 +70,62 @@ def from_file(cls, catalog_file, verbose=False):
     def _prepare_table(self):
         table = deepcopy(self._raw_catalog)
 
+
         # TODO: does the WLDCatalog require the 'ra' and 'dec' columns
         # convert ra dec from degrees to arcsec in catalog.
-        if "ra" in table.columns:
-            table["ra"] *= 3600
-        if "dec" in table.columns:
+        if "ra" in table:
```

Comment:
this will raise a `FutureWarning`, you should keep `table.columns` or `table.colnames` if you wish.

Reply:
```suggestion
        if "ra" in table.colnames:
```